api/src/test/java/org/pmiops/workbench/interactiveanalysis/InteractiveAnalysisServiceTest.java

```diff
@@ -156,17 +156,14 @@ public void testLocalize_jupyter_playground() {
   @Test
   public void testLocalize_RStudio_editMode() {
     String editDir = "";
-    String playgroundDir = "workspaces_playground";
     List<String> notebookLists = List.of("foo.Rmd");
     StorageLink expectedStorageLink =
         new StorageLink()
             .cloudStorageDirectory(NOTEBOOK_DIR)
             .localBaseDirectory(editDir)
-            .localSafeModeBaseDirectory(playgroundDir)
```

Comment:
[this](https://github.com/DataBiosphere/welder/blob/7da52558140408e69a4f3128510f4129178d55a6/core/src/main/scala/org/broadinstitute/dsp/workbench/welder/model.scala#L105) is indeed optional!